fsdevtools-cli/src/main/java/com/espirit/moddev/cli/commands/scheduleEntry/ScheduleEntryStartCommand.java

```diff
@@ -0,0 +1,131 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+/**
+ * This Command can start a project schedule entry.
+ *
+ * @author e-Spirit AG
+ */
+@Command(name = "start", groupNames = {"scheduleentry"}, description = "Starts a Project schedule entry.")
+
+@Examples(examples =
+        {
+                "scheduleentry start -sen \"myScheduleEntry\""
+        },
+        descriptions = {
+                "Starts the schedule entry.",
+        })
+@SuppressWarnings("squid:S1200")
+public class ScheduleEntryStartCommand extends SimpleCommand<SimpleResult<Boolean>> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ScheduleEntryStartCommand.class);
+
+    @Option(name = {"-sen", "--scheduleEntryName"}, description = "The schedule entry name.")
+    private String scheduleEntryName = "";
+
+    @Option(name = {"-wff", "--waitForFinish"}, description = "Wait for the task to finish. (default is true)")
+    private String waitForFinish = "true";
+
+    @Override
+    public SimpleResult<Boolean> call() {
+
+        try (Connection connection = create()) {
+            connection.connect();
+
+            if (!connection.isConnected()) {
+                throw new IllegalStateException("Connection is null or not connected!");
+            }
+
+            return executeScheduleEntry(connection);
+        } catch (IOException | AuthenticationException | MaximumNumberOfSessionsExceededException | IllegalArgumentException | IllegalStateException e) {
+            LOGGER.error("Can't connect to the FirstSpirit Server", e);
+            return new SimpleResult<>(e);
+        } catch (ScheduleEntryRunningException e) {
+            LOGGER.error("Can't execute schedule entry", e);
+            return new SimpleResult<>(e);
+        }
+    }
+
+    private SimpleResult<Boolean> executeScheduleEntry(Connection connection) throws ScheduleEntryRunningException {
+        ServicesBroker servicesBroker = connection.getBroker().requireSpecialist(ServicesBroker.TYPE);
+        AdminService adminService = servicesBroker.getService(AdminService.class);
+        Project project = adminService.getProjectStorage().getProject(super.getProject());
+        ScheduleEntry scheduleEntry = adminService.getScheduleStorage().getScheduleEntry(project, getScheduleEntryName());
+
+        if (scheduleEntry == null) {
+            LOGGER.error("Can't find schedule entry. Nothing to execute");
+            return new SimpleResult<>(false);
+        }
+
+        LOGGER.info(String.format("Found schedule %s on project %s", getScheduleEntryName(), super.getProject()));
+        LOGGER.info("Execute schedule");
+        ScheduleEntryControl scheduleEntryControl = scheduleEntry.execute();
+        if (isWaitForFinish()) {
+            LOGGER.info("Wait for the task to finish.");
+            scheduleEntryControl.awaitTermination();
+        }
+
+        return new SimpleResult<>(true);
```

Comment:
I don't think SimpleResult and true do fit here - the return should be a custom result type with at least the log output of the schedule and it's success/failure, don't you think?

Reply:
Hi, this is correct. For now i have no time to edit and test this. I will send a new pull request, if i ready.